unix/tx/TXWindow.cxx

```diff
@@ -217,6 +217,10 @@ void TXWindow::getColours(Display* dpy, XColor* cols, int nCols)
       XFreeColors(dpy, cmap, &p, 1, 0);
     }
   }
+
+  delete [] got;
+  delete [] shared;
+  delete [] usedAsNearest;
```

Comment:
Compilers support dynamic arrays these days, so I would prefer a change to that. E.g. `bool got[nCols];`.